product_contract/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import test_product_template
+from . import test_sale_order
```

Comment:
Preferences => File Saving => Ensure File End With EOL Marker (assuming you're still using Komodo)